docs/rules/extensions.md

```diff
@@ -86,6 +86,30 @@ import express from 'express/index.js';
 import * as path from 'path';
 ```
 
+The following patterns are considered problems when configuration set to "ignorePackages":
```

Comment:
let's add an example of `require('foo/bar.js')` still being required with both "always" and "ignorePackages" set.

Reply:
this still needs an example, and a matching test

Reply:
Thanks for the reminder, @ljharb. I beleive [these commits](https://github.com/millerized/eslint-plugin-import/pull/1/files/00062fbdcf6e3eaf492fcbb4020e9b35239fdf05..59b9dfd2f52cec10b802d2d8f355552cb4002afb) should suffice. lmk what you think.